packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -184,14 +182,13 @@ export class InputTimePicker
 
   @Watch("open")
   openHandler(): void {
-    onToggleOpenCloseComponent(this);
-
     if (this.disabled || this.readOnly) {
       this.open = false;
       return;
     }
 
-    this.reposition(true);
+    // we set the property instead of the attribute to ensure popover's open/close events are emitted properly
+    this.popoverEl.open = this.open;
```

Comment:
What if the time picker is open by default? Would this still happen? I'm not seeing anything set the popover to be open except in this watcher which wouldn't run initially.

Reply:
Ah, great catch. 🎣 Will fix!

Reply:
Fixed! Added a test for this as well. ✨🔧✨